src/flatpak/mod.rs

```diff
@@ -125,28 +120,28 @@ pub struct SpawnOptions {
     /// A list of file descriptor for files inside the sandbox that will be
     /// exposed to the new sandbox, for reading and writing.
     #[zvariant(rename = "sandbox-expose-fd")]
-    sandbox_expose_fd: Option<Vec<Fd>>,
+    sandbox_expose_fd: Vec<Fd<'f>>,
```

Comment:
there is a reason why all those are optional, zbus's SerializeDict doesn't provide a skip_serializing_if functionnality and the portals would explode.

Reply:
Ah ok, I thought None was the same as an empty vec

Reply:
Using `Option<Vec<Fd<'f>>>` now.